Add fusion tip about AWS storage config (#5672)
nextflow-io/nextflow: docs/fusion.md

the option "Use Fast storage" when creating the Batch compute environment.
:::

> :::{tip}

Reviewer: I think it's important to stress that it's a good alternative but not as good as NVMe disk.

"While not as fast as NVMe storage, an EBS gp3 volume with a throughput of 325 MiB/s or more and a size of 100 GiB or more provides sufficient performance for many workloads on your compute node"

> In alternative to setting up a NVMe storage in you compute node, you can use an EBS volume type `gp3` selecting
> 325 MiB/s (or more as throughput) and 100 GiB (or more as size). 

Reviewer 2: ```suggestion
As an alternative to configuring NVMe storage on your compute node, you can use an EBS `gp3` volume with a throughput of 325 MiB/s or more and a size of 100 GiB or more.
```
Please check meaning is retained.

Reviewer: Looks good to me
